python-requirements.okd

```diff
@@ -0,0 +1,4 @@
+ironic
+proliantutils
+pyghmi
+python-scciclient
```

Comment:
```suggestion
git+https://github.com/openshift/openstack-ironic
proliantutils>=2.16.3
python-scciclient>=0.17.0
```

This is the minimum version of what I was talking about in my comment. It makes sure you download the version that is a "midstream" for OpenShift rather than whatever ends up published on PyPI (which may not be compatible with the current OpenShift main).